examples/twittermap/web/conf/application.conf

```diff
@@ -115,3 +115,6 @@ us.city.path = "/public/data/city.sample.json"
 
 tweet.url = "http://twitterMapIndex-search-proxy.herokuapp.com/search/tweets?q=%s"
 
+# Pointmap parameters
```

Comment:
Add comments to explain the purpose of these two parameters.

Reply:
In addition, you need to explain what these parameters do. Also, if the default value is used in case no parameters are provided, we can comment the following two parameters (e.g., querySliceMills and cacheThreshold).